contracts/SourceOfRandomnessHistory.cdc

```diff
@@ -0,0 +1,17 @@
+pub contract SourceOfRandomnessHistory {
+
+    /// Canonical storage path for the SourceOfRandomnessHistory.Heartbeat resource.
+    pub let HeartbeatStoragePath: StoragePath
+
+    pub resource Heartbeat {
+        pub fun heartbeat(randomSourceHistory: [UInt8]) {
```

Comment:
Are we now receiving the `randomeSourceHistory` from a transaction submitted by the service account instead of exposing the SoR via specialized method?